web/e2e/page-objects/flows/manageFlows.ts

```diff
@@ -24,6 +23,10 @@ export class ManageFlows extends AppPage {
     return await button.click();
   }
 
+  get flowDialogBox() {
+    return element(by.css(".mat-dialog-container"));
```

Comment:
is this supposed to be element or class? look at stepDialogBox above

Reply:
Both work "<mat-dialog-container aria-modal="true" class="mat-dialog-container ng-tns-c25-10 ng-trigger ng-trigger-dialogContainer ng-star-inserted" tabindex="-1""